Source/Events/OnAuctionHouseShow.lua

```diff
@@ -5,5 +5,7 @@ function Auctionator.Events.OnAuctionHouseShow()
     Auctionator.State.AuctionatorFrame = CreateFrame("FRAME", "AuctionatorAHFrame", AuctionHouseFrame, "AuctionatorAHFrameTemplate")
   end
 
-  FrameUtil.RegisterFrameForEvents(Auctionator.State.AuctionatorFrame, { "AUCTION_HOUSE_SHOW", "AUCTION_HOUSE_CLOSED" })
+  FrameUtil.RegisterFrameForEvents(Auctionator.State.AuctionatorFrame, { "AUCTION_HOUSE_SHOW", "AUCTION_HOUSE_CLOSED", "AUCTION_HOUSE_THROTTLED_SYSTEM_READY" })
```

Comment:
Can you wrap this for readability:
```lua
FrameUtil.RegisterFrameForEvents(
  Auctionator.State.AuctionatorFrame,
  {
    "AUCTION_HOUSE_SHOW",
    "AUCTION_HOUSE_CLOSED", 
   "AUCTION_HOUSE_THROTTLED_SYSTEM_READY"
  }
)
```

Reply:
Didn't mean to include this change in the PR, reverted it